assets/src/setup/pages/choose-reader-theme/theme-card.js

```diff
@@ -32,13 +34,24 @@ export function ThemeCard( { description, homepage, screenshotUrl, slug, name }
 
 	const id = `theme-card__${ slug }`;
 
+	const truncatedDescription = useMemo( () => {
```

Comment:
Any reason why this could not be accomplished via CSS?

Reply:
Unfortunately CSS `text-overflow:ellipsis` only supports truncating when it is a single line. It doesn't support eliding multiple lines. See also https://css-tricks.com/line-clampin/

Reply:
That being said, @johnwatkins0 could an approach from that CSS Tricks article be used so that we can retain the original text? Should the full description be added as a tooltip of the element?

Reply:
Actually, there is [`-webkit-line-clamp`](https://dev.to/temmietope/how-to-truncate-multi-line-string-with-pure-css-4fc9). Despite its name, it _is_ supported by Firefox. The fallback behavior would be for the text to just be hidden without the ellipsis, right? 

See also https://dev.to/temmietope/how-to-truncate-multi-line-string-with-pure-css-4fc9

It's the `-webkit-line-clamp` property is supported by all current browsers (95% global usage): https://caniuse.com/#feat=css-line-clamp

So it seems safe to use!

Reply:
Gonna try that.

Reply:
> Actually, there is -webkit-line-clamp.

Was about to suggest that :smile:.

Reply:
Not bad:

<img width="1054" alt="Screen Shot 2020-06-15 at 4 27 13 PM" src="https://user-images.githubusercontent.com/9020968/84707917-29504a80-af25-11ea-9f07-f72f5d65ba37.png">

Reply:
Pushed the line clamp.

As for the tooltip, a tooltip would work, but the "Learn more" link opens in a new tab and has the full description. Also, with a tooltip we'd have to figure out whether the description is truncated, which would be easy with the JS-based truncation but not so much with the CSS version. It might be worth exploring (cc @jwold) and if worth it we can go back to doing it in JS.

Reply:
Are we suggesting a tooltip that shows the remainder after the ellipsis? If so I would think it would behave something like this: 

<img width="1139" alt="My Sketches - 2020-06-16 07 40 44" src="https://user-images.githubusercontent.com/3411173/84789984-ade2ad80-afa5-11ea-8c9d-be7a32ba24da.png">

If you hover anywhere over the paragraph description (screen 1), your cursor will display a tooltip (screen 2).

If that's what we're suggesting I don't think it's necessary because:

1. Some of the descriptions are very long, multiple paragraphs. A tooltip won't solve that by showing enough information.
2. We have the learn more link, so they're just a click away from reading the whole thing themselves.